tests/silent-payment.test.ts

```diff
@@ -28,33 +46,34 @@ it("smoke test", () => {
 /* Sending tests from the BIP352 test vectors */
 tests.forEach((testCase, index) => {
   // Prepare the 'inputs' array
-  const inputs = testCase.given.outpoints.map((outpoint, idx) => ({
-    txid: outpoint[0],
-    vout: outpoint[1],
-    WIF: ECPair.fromPrivateKey(Buffer.from(testCase.given.input_priv_keys[idx][0], "hex")).toWIF(),
-    isTaproot: testCase.given.input_priv_keys[idx][1],
-  }));
+  testCase.sending.forEach(sending => {
+      const utxos = sending.given.vin.map((input) => ({
+        txid: input.txid,
+        vout: input.vout,
+        WIF: ECPair.fromPrivateKey(Buffer.from(input.private_key, "hex")).toWIF(),
+        utxoType: getUTXOType(input) as UTXOType,
+      }));
+      const noEligibleUtxos = utxos.every(utxo => utxo.utxoType === 'non-eligible');
 
-  // Prepare the 'recipients' array
-  const recipients = testCase.given.recipients.map((recipient) => ({
-    silentPaymentCode: recipient[0],
-    value: recipient[1],
-  }));
+      // Prepare the 'recipients' array
+      const recipients = sending.given.recipients.map((recipient) => ({
+        silentPaymentCode: recipient,
+        value: 1,
```

Comment:
1 sat?

Reply:
ah, just threw a value in here, but I guess since its `value?` I could just not set the field?

Reply:
you can leave it, nbd.
fyi, the contract with `coinselect` library we use is that if no `value` is specified, that means values are split evenly across destinations (or all coins are sent in case of a single destination )